action.yaml

```diff
@@ -27,28 +27,41 @@ inputs:
   delete-branch:
     description: 'Delete PR branch after merge'
     required: false
-    default: true
+    default: 'true'
   branch-prefix:
     description: 'The prefix for the name of the feature branch'
     required: false
     default: 'automated-code-release-'
+  working-directory:
```

Comment:
Since this is being used, does this not need a default?

Reply:
it doesn't need a default. if you don't specify a value, then it's like the option is never specified to the nested action, so the default is whatever the nested actions default is. the cycjimmy/semantic-release-action doesn't have a default for me to copy explicitly, so I just left it blank